src/computer.h

```diff
@@ -116,6 +116,8 @@ class computer
     private:
         // Difficulty of simply logging in
         int security;
+        // Date of next attempt
+        long next_attempt = 0;
```

Comment:
In my time ```int``` was 2^15 **heavy cough*\* 

Thanks for clarifications, I'm going to change this to ```int``` a little bit later.